osquery/tables/applications/browser_utils.cpp

```diff
@@ -74,9 +78,31 @@ void genExtension(const std::string& uid,
     }
   }
 
+  // Fetch the permission array from the manifest file
+  std::string permission_list;
+
+  const auto& perm_array_obj = tree.get_child_optional(kExtensionPermissionKey);
+  if (perm_array_obj) {
+    const auto& perm_array_contents = perm_array_obj.get();
+    for (auto it = perm_array_contents.begin(); it != perm_array_contents.end();
+         ++it) {
+      const auto& perm_obj = *it;
+      const auto& permission =
+          perm_obj.second.get_value_optional<std::string>();
+      if (permission) {
+        permission_list.append(*permission);
+      }
+
+      if (std::next(it, 1) != perm_array_contents.end()) {
```

Comment:
Actually it's not correct anymore.(after changing get_value to get_value optional
Either we should use vector and join or
```
 if (permission) {
       if(permission_list.size() != 0){
          permission_list.append(", ");
       }
        permission_list.append(*permission);
 }
```